src/components/table/_table.scss

```diff
@@ -110,6 +110,7 @@
     // Default `max-height` before a scrollbar will show
     // We don't use `height` as table could be shorter than this value
     max-height: $b-table-sticky-header-max-height;
+    border-top: $table-border-width solid $table-border-color;
```

Comment:
This is assuming users haven't set a `table-{variant}` class on the table (via the `table-variant` pop), which sets the color of the borders as well. this would override that.

Reply:
probably better written as:

```scss
    border-top-width: $table-border-width;
```

To just affect the border width only, and not the color of the border

Reply:
Otherwise border colors will need to be generated for each possible variant color.